pymc/tests/test_transforms.py

```diff
@@ -220,7 +226,7 @@ def transform_params(x, z=a, y=b):
         close_to_logical(vals < b, True, tol)
 
 
-@pytest.mark.skipif(aesara.config.floatX == "float32", reason="Test fails on 32 bit")
+@pytest.mark.xfail(reason="This test produces infinite values using aeppl")
```

Comment:
Does that mean the circular transform is broken?

Reply:
This was discussed during the call. We should probably open an issue regarding this.